lib/elasticsearch_client.py

```diff
@@ -131,7 +132,11 @@ def save_alert(self, body, index='alerts', doc_type='alert', doc_id=None, bulk=F
         return self.save_object(index=index, doc_type=doc_type, body=body, doc_id=doc_id, bulk=bulk)
 
     def save_event(self, body, index='events', doc_type='event', doc_id=None, bulk=False):
-        return self.save_object(index=index, doc_type=doc_type, body=body, doc_id=doc_id, bulk=bulk)
+        if type(body) is str:
```

Comment:
While we do this logic in save_object as well, there are cases in our code base (the mq/esworker_sns_sqs.py for example), where we call the save_event with the body being a string, and not json.

We can either do this parsing logic in save_event, or in the Event class __init__ function, but unfortunately I think we're going to have to have these checks in both save_object, and save_event.